solana/crates/solbox/src/lib.rs

```diff
@@ -269,15 +300,10 @@ impl<'a> Context<'a> {
         pda: &AccountInfo,
         seeds: &[&[u8]],
     ) -> Result<PdaData<T, D>> {
-        unsafe {
```

Comment:
Just curious: what changed to allow you to get rid of all these unsafe blocks?

Reply:
The pinocchio authors decided that it's no longer actually unsafe in the new versions - https://github.com/anza-xyz/pinocchio/pull/180